en/peripherals/rtk_gps_cuav_c-rtk.md

```diff
@@ -15,29 +16,27 @@ RTK setup and use on PX4 via _QGroundControl_ is largely plug and play \(see [RT
 
 ## Wiring and Connections
 
-C-RTK GPS with 6-pin connector and 4-pin connector, 6-pin connector can be plugged directly into [Pixhack v3](http://doc.cuav.net/gps/c-rtk/en/Instructions.html) GPS UART port, If you only need rtk gps, 4-pin connector does not need to be connected to flight control.
+C-RTK GPS comes with a cable that terminates in a 6-pin connector and 4-pin connector that are compatible with [Pixhack v3](http://doc.cuav.net/gps/c-rtk/en/Instructions.html).
+The 6-pin connector provides the interface for RTK GPS, and should be connected to the flight controller's GPS port.
+The 4-pin connector is an m8n (standard) GPS interface that is intended for (optional) use as a second GPS. 
+
+> **Tip** At time of writing PX4 does not yet fully support a second GPS. The 4-pin port need not be connected.
 
-The Pixhawk 3 Pro and Pixracer have 6-pin GPS port connectors. For these controllers, you can modify the order of the GPS cables \(shown below\).![C-RTK_6PIN](/assets/rtk_cuav_c-rtk_to_6pin_connector.jpg)
+<img src="../../assets/hardware/gps/rtk_cuav_c-rtk_to_6pin_connector.jpg" width="500px" title="C-RTK_6PIN" />
```

Comment:
FYI, needed to use this format in order to change width without having to change the actual image.